cli/azd/cmd/monitor.go

```diff
@@ -89,7 +89,13 @@ func (m *monitorAction) Run(ctx context.Context, cmd *cobra.Command, args []stri
 	}
 
 	resourceManager := infra.NewAzureResourceManager(ctx)
-	resourceGroups, err := resourceManager.GetResourceGroupsForDeployment(ctx, env.GetSubscriptionId(), env.GetEnvName())
+	resourceGroupsResources, err := resourceManager.GetResourceGroupsForEnvironment(ctx, &env)
+	resourceGroups := []string{}
+
+	for _, value := range resourceGroupsResources {
+		resourceGroups = append(resourceGroups, value.Name)
```

Comment:
Would it be better to avoid copying all values to a new struct and just add `.Name` below (where `resourceGroup` is used) ???